.ci/azure-pipelines/env.yml

```
@@ -114,7 +114,7 @@ jobs:
         PLATFORM: x86
         TAG: windows2022-x86
         GENERATOR: "'Visual Studio 16 2019' -A Win32"
-        VCPKGCOMMIT: 8eb57355a4ffb410a2e94c07b4dca2dffbee8e50
+        VCPKGCOMMIT: 01f602195983451bc83e72f4214af2cbc495aa94
```

```suggestion
        VCPKGCOMMIT: f7423ee180c4b7f40d43402c2feb3859161ef625
```
There was a [problem with the hashes with the boost port](https://github.com/microsoft/vcpkg/commit/a8954b904ad2a6939ecd8fc213e87702fa1243ea), seems like we have to use [release 2024.06.15](https://github.com/microsoft/vcpkg/releases/tag/2024.06.15) instead.